File: CS/ExportReportDemoEF/ExportReportDemoEF.Module/BusinessObjects/Employee.cs
using DevExpress.Persistent.Base;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ExportReportDemoEF.Module.BusinessObjects
{
    [DefaultClassOptions]
    public class Employee
    {
        [Browsable(false)]
        public int ID { get; set; }
        [VisibleInListView(false)]
        public string FirstName { get; set; }
        [VisibleInListView(false)]
        public string LastName { get; set; }
        public string FullName
        {
            get { return string.Format("{0} {1}", FirstName, LastName); }
        }
        public string Position { get; set; }
    }

}

File: VB/ExportReportDemoEF/ExportReportDemoEF.Module/BusinessObjects/Employee.vb
Imports DevExpress.Persistent.Base
Imports System.ComponentModel

Namespace ExportReportDemoEF.Module.BusinessObjects

    <DefaultClassOptions>
    Public Class Employee

        <Browsable(False)>
        Public Property ID As Integer

        <VisibleInListView(False)>
        Public Property FirstName As String

        <VisibleInListView(False)>
        Public Property LastName As String

        Public ReadOnly Property FullName As String
            Get
                Return String.Format("{0} {1}", FirstName, LastName)
            End Get
        End Property

        Public Property Position As String
    End Class
End Namespace
